### PR TITLE
Add support for reading the PM table on Phoenix APUs

### PR DESCRIPTION
**Pull request by patrickschur**
Add support for reading the PM table on Phoenix APUs (solves #255 and #246)

### CODE REVIEW

File: lib/api.c [hunk 1]
```diff
@@ -137,6 +137,7 @@ int request_table_ver_and_size(ryzen_access ry)
 	case FAM_LUCIENNE:
 	case FAM_CEZANNE:
 	case FAM_REMBRANDT:
+	case FAM_PHEONIX:
```
Comment (ruineka) [inline lib/api.c:140]: Did you mean `PHOENIX` here and not `PHEONIX`?

File: lib/api.c [hunk 2]
```diff
@@ -212,9 +215,12 @@ int request_table_addr(ryzen_access ry)
...
 
-	if (ry->family == FAM_REMBRANDT) {
+	switch (ry->family)
+	{
+	case FAM_REMBRANDT:
+	case FAM_PHEONIX:
 		ry->table_addr = (uint64_t) args.arg1 << 32 | args.arg0;
-	} else {
+	default:
 		ry->table_addr = args.arg0;
```
Comment (ruineka) [inline lib/api.c:224]: The problem is here. I changed the default to this `ry->table_addr = (uint64_t) args.arg1 << 32 | args.arg0;` and it works.

File: lib/api.c (hunk 1 above)
Comment (patrickschur) [reply lib/api.c:140]: Autocompletion... I also noticed that. But the typo wasn't introduced by me.

Comment (patrickschur) [reply lib/api.c:140]: I will fix the typo later.

File: lib/api.c (hunk 2 above)
Comment (patrickschur) [reply lib/api.c:225]: Yes, thanks for that. I just forgot the break statement.

**Comment by FlyGoat** [comment]
LGTM, now doing some further test.
I was trying to diff new PMTable on my machine, this PR saved me from this painful job :-)

**Comment by FlyGoat** [comment]
Extensively tested on my system.
LGTM, Thanks!
Will Squash merge and release ASAP.

**Pull request merged**